vsintegration/src/FSharp.Editor/Common/LanguageService.fs

```diff
@@ -291,6 +291,9 @@ and
                 do! setupProjectsAfterSolutionOpen() 
             }
         setupProjectsAfterSolutionOpen() |> Async.StartImmediate
+
+        let theme = package.ComponentModel.DefaultExportProvider.GetExport<ISetThemeColors>().Value
```

Comment:
I wonder if there is a better way to reliably instantiate it. I'm not an expert in MEF :(